movePlayerScript.js

```diff
@@ -0,0 +1,49 @@
+var storedKeyPress = 65;
+
+$(document).keydown(function(e) {
+    var moveDistance = "+=10px";
+    moveLeg(e, moveDistance, "#playerProfile")
+    var positionOfMan = parseInt($("#playerProfile").css('left'))
+    var positionOfFinish = parseInt($("#pumpkinFinishLine").css('left'))
+    finishLine(positionOfMan, positionOfFinish)
+})
+
+// This function moves a html element across the screen.
+// For parameters it takes the event "e" from the event listener,
+// the distance you want the html element to move
+// and the id of the html element you want to move.
+// It changes the storedKeyPress variable to prevent the same key being pressed twice.
+function moveLeg(e, distance, htmlId) {
```

Comment:
moveLeg doesn't work as a function name any more
htmlId isnt being passed in as anything else so just hardcode it in the function and dont pass it in as a param